keps/sig-auth/3257-cluster-trust-bundles/README.md

```diff
@@ -739,15 +813,36 @@ in back-to-back releases.
 
 ### Upgrade / Downgrade Strategy
 
-At present, there are no upgrade / downgrade concerns.  The ClusterTrustBundle
-feature gate controls overall availability of the feature.
+The feature is gated by these feature flags:
+- kube-apiserver
+    - `ClusterTrustBundle` controls availability of the ClusterTrustBundle API and
+      presence of the relevant rules in cluster roles for the kubelet and KCM.
+    - `ClusterTrustBundleProjection` controls availability of the `ClusterTrustBundle`
+      projected volume source in the Pod API.
+- kubelet, KCM
+    - watch API discovery for the ClusterTrustBundle API and enable the feature
+      internally once the API gets available.
+    - to disable the feature after the `ClusterTrustBundle` feature gate disables
+      the API availability, it is necessary to restart both the kubelet and KCM
+
+The feature is enabled by the kube-apiserver feature gates. Aside from enabling the API, the `ClusterTrustBundle`
```

Comment:
Are there other APIs that span KAS and kubelet like CTB that we can learn from in terms of how the feature gates are expected to work?